Bloods Rc plugin/minor changes.

Zeus-Xora opened the pull request:
Updated repair with Lunar to include space key presses and prevent failure to repair.

Updated Activity to include my set intensity.

Updated PluginDescriptor to include myself.

True Blood Rc Plugin

Zeus-Xora committed:
Added myself to PluginDescriptor for future plugins

Zeus-Xora committed:
Update Rs2Magic.java

updated repair pouches so that it does not become stuck

Zeus-Xora committed:
Update Activity.java

Added set intensity for my Rc plugin

Zeus-Xora committed:
BloodsPlugin

Bloods plugin

chsami reviewed (changes requested):
Our philosophy is simple: less config is better for the end user.

Rather than asking users to tweak settings or define behavior manually, we push the complexity into the script itself. It should just work™ out of the box, with sensible defaults and smart handling behind the scenes.

This update simplifies the user experience by reducing the number of required config options while maintaining full control and flexibility for developers.

No clutter. No overthinking. Just clean automation.

chsami commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/frosty/bloods/BloodsConfig.java:
```diff
@@ -0,0 +1,64 @@
+            position = 1,
+            section = settingsSection
+    )
+    default Teleports teleports() {return Teleports.CRAFTING_CAPE;
```
config that can be calculated based on inventory/equipped item?

chsami commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/frosty/bloods/BloodsConfig.java:
```diff
@@ -0,0 +1,64 @@
+            position = 2,
+            section = settingsSection
+    )
+    default HomeTeleports homeTeleports() { return HomeTeleports.CONSTRUCTION_CAPE;}
```
config that can be calculated based on inventory/equipped item?

chsami commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/frosty/bloods/BloodsConfig.java:
```diff
@@ -0,0 +1,64 @@
+            position = 4,
+            section = settingsSection
+    )
+    default boolean useDramenStaff() {
```
config that can be calculated based on inventory/equipped item?

Zeus-Xora committed:
Blood rc plugin

Zeus-Xora committed:
Blood rc plugin

Script now handles where to bank.

Removed CastleWars and added Ferox Enclave.
Supports run energy at Ferox.

Added Ardougne cloak teleport to use fairy ring there if not using PoH

Zeus-Xora committed:
Removed Dramen staff config to read Achievement diary information. If Lumby Elite diary is not complete, we will auto withdrawandequip Dramen staff

Zeus-Xora committed:
Fixed the Dramen staff check.

Fixed walk to location to ensure agility obstacle is handled.
